solvcon/mesh.pyx

```diff
@@ -221,6 +221,17 @@ cdef class Mesh:
             if vals[0] != vals[1]:
                 msgs.append('body(%d,%d)' % vals)
             if msgs:
+                f = open("/tmp/solvcon.debug.log", "a")
+                msgs.append("\n" + str(dir(_table)) + "\n")
+                msgs.append(str(_cnda))
+                msgs.append(str(_table._ghostaddr))
+                msgs.append(str(_table._bodyaddr))
+                msgs.append(str(_table.nghost))
+                msgs.append(str(_table.nbody))
+                #msgs.append(str(_table._full))
+                #msgs.append(str(_table._body))
+                f.write(','.join(msgs) + "\n")
+                f.close()
                 tmpl = '%s array mismatch: %s'
```

Comment:
Yes, it seems ``Table.__init__()`` is skipped totally.

Your print-out is informative and should be added to the code.  The error message would be more readable if it is like "_table._nda = None, ..., _table.nghost = -1, _table.nbody = -1", and it would save you a lot of explanation.